cfme/tests/services/test_different_dialogs_in_catalogs.py

```diff
@@ -52,11 +51,9 @@ def tagcontrol_dialog(appliance):
 
 
 @pytest.yield_fixture(scope="function")
-def catalog():
+def catalog(appliance):
     catalog = "cat_" + fauxfactory.gen_alphanumeric()
-    cat = Catalog(name=catalog,
-                  description="my catalog")
-    cat.create()
+    cat = appliance.collections.catalogs.create(name=catalog, description="my catalog")
```

Comment:
This change needs to be made in these places too : 
https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/services/test_catalog.py#L19
https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/services/test_provision_stack.py#L52
https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/services/test_dynamicdd_dialogelement.py#L74
https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/ansible/test_embedded_ansible_services.py#L110

Reply:
Thanks! Not sure how I missed those.